### PR TITLE
dos2unix: add French translation

### PR DESCRIPTION
**Pull request by pascal-brand38**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/linux/dos2unix.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# dos2unix
+
+> Remplace les fins de lignes de style DOS par des fins de lignes de style Unix.
+> Remplace CRLF par LF.
+> Voir également `unix2dos`, `unix2mac`, et `mac2unix`.
+> Plus d'informations : <https://manned.org/dos2unix>.
+
+- Remplace les fins de lignes d'un fichier:
```
Comment (noraj) [inline pages.fr/linux/dos2unix.md:8]:
```suggestion
- Remplace les fins de lignes d'un fichier :
```

File: pages.fr/linux/dos2unix.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+`dos2unix {{chemin/vers/fichier}}`
+
+- Créer une copie avec des fins de lignes de type Unix:
```
Comment (noraj) [inline pages.fr/linux/dos2unix.md:12]:
```suggestion
- Crée une copie avec des fins de lignes de type Unix :
```

File: pages.fr/linux/dos2unix.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+`dos2unix -n {{chemin/vers/fichier}} {{chemin/vers/nouveau_fichier}}`
+
+- Affiche les informations d'un fichier:
```
Comment (noraj) [inline pages.fr/linux/dos2unix.md:16]:
```suggestion
- Affiche les informations d'un fichier :
```

File: pages.fr/linux/dos2unix.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+`dos2unix -i {{chemin/vers/fichier}}`
+
+- Conserve/Ecrit/Supprime la marque d'ordre des octets (BOM):
```
Comment (noraj) [inline pages.fr/linux/dos2unix.md:20]:
```suggestion
- Conserve/Ecrit/Supprime la marque d'ordre des octets (BOM) :
```

**Review by noraj — CHANGES_REQUESTED** [review]
1. Concernant la typographie, il doit y avoir un espace avant le `:`
2. Toutes les phrases doivent être au présent de l'indicatif plutôt qu'à l'infinitif

**Review by noraj — APPROVED** [review]
LGTM now

**Review by spageektti — APPROVED** [review]

**Pull request merged**